server/app.ts

```diff
@@ -35,7 +38,9 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
   nunjucksSetup(app, services.applicationInfo)
+  app.use(setupSmokeTest(services.smokeTestStore))
   app.use(setUpAuthentication())
+  app.use(auth.authenticationMiddleware(tokenVerifier, services.smokeTestStore))
```

Comment:
As discussed, would be worth seeing if this can be left where it was as it's unclear why it needs to move here.